data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -73,49 +68,38 @@ then
 	touch cvo-bootstrap.done
 fi
 
-# during initial operator rollout phase this logic allows us to deploy the operator via CVO
-# in an `Unmanaged` no-op state. after all of the pieces have merged and the operator is
-# deemed stable we can remove this logic and the operator will be `Managed` by default.
-if [ ! -z "$CLUSTER_ETCD_OPERATOR_MANAGED" ]
+# TODO: host-etcd endpoint rendered by cluster-etcd-operator
```

Comment:
can we add a link to etcd JIRA or etcd-operator github issue to track the TODO..